ebpfsvc/svcmain.cpp

```diff
@@ -12,23 +12,23 @@ SERVICE_STATUS_HANDLE ebpf_service_status_handle;
 HANDLE ebpf_service_stop_event_handle = nullptr;
 
 void WINAPI
-service_control_handler(DWORD ctrl);
+service_control_handler(_In_ unsigned long ctrl);
```

Comment:
`_In_` should not be used on pass-by-value types, only pointer types.  So this is only ok if it is required to compile to match a prototype in a windows header that does it wrong.  Same in line 19, etc.